playbooks/roles/unbound/defaults/main.yml

```diff
@@ -0,0 +1,4 @@
+# Set the zone forwardnig configuration in the 'forward_zones' list. Each entry will require a 'name' key and
+# a 'forward-addr'/'forward-host' key with appropriate values
```

Comment:
This comment looks like it should be above` forward_zones`, not `unbound_config_dir`? And then we can also add some spacing.